run_action.py

```diff
@@ -719,7 +730,12 @@ def main():
     warning_comment_prefix = (
         ":warning: `Clang-Tidy` found issue(s) with the introduced code"
     )
-    single_comment_marker = ":warning:"
+    single_comment_markers = {
+        "Error": ":x:",
+        "Warning": ":warning:",
+        "Remark": ":speech_balloon:",
+        "fallback": ":grey_question:",
+    }
```

Comment:
There's a pylint error about having too many local variables in this function (16 instead of max 15).
It's supposedly a bad practice as an indicator of the function being too complex. :shrug: 

I don't 100% agree, but the easy fix would be to move `single_comment_markers` to the global scope.

Reply:
Personally, I find such restrictions rather arbitrary (why having 15 local variables is fine but 16 is already "a lot"?) and I regard it as a hint at best. If I find the hint inappropriate, I simply disable it through the appropriate comment, which I recommend doing in this particular case.

Reply:
No strong opinions from my side on this, it feels like a rather opinionated "error".

Reply:
I factored out the reordering, so this is not an issue anymore.